forth_src/io.fs

```diff
@@ -0,0 +1,79 @@
+require open
+
+\ Set input device to open file #
+\ returns 0 on success, file # on error
+code chkin? ( file# -- file# result )
+w stx,
+lsb lda,x tax, \ x = file#
+$ffc6 jsr, \ CHKIN
++branch bcs, \ carry set = error
+0 lda,# \ A is only valid on error
+:+
+w ldx, dex, 
+lsb sta,x
+0 lda,# msb sta,x
+;code
+
+\ Set output device to open file #
+\ returns 0 on success, file # on error
+code chkout? ( file# -- file# result )
```

Comment:
This comment looks wrong, should it be "\ result=0 on success, or OPEN error code" ?